snap/snapcraft.yaml

```diff
@@ -17,16 +17,26 @@ slots:
     name: se.sjoerd.Graphs
 
 parts:
+  blueprint-compiler:
+    source: https://gitlab.gnome.org/jwestman/blueprint-compiler.git
+    source-tag: 'v0.8.1'
+    plugin: meson
+    meson-parameters:
+      - --prefix=/usr
+    override-prime: |
+      echo 'Skip'
```

Comment:
I was just curious what the override-prime does?
It seems like it just prints a "Skip" in this step, so what was the purpose of these two lines? (line 26 and line 27)

Reply:
Snap has four steps to build, 
1. Pull
2. Build
3. Stage
4. Prime

Pull: Pulls all the repos and listed repo packages and saves the repo packages in `$CRAFT_STAGE` folder
Build: Builds it
Stage: Stages the build from every part to the `$CRAFT_STAGE` path
Prime: Copies those files which are said to be primed, into the `$CRAFT_PRIME` folder

Now, here, I am saying the builder to not prime the files of `blueprint-compiler` part. So, I am overriding the prime, and just making an arbitrary command to run.

Reply:
Thank you for  the explanation, I am new to Snap packaging so I was just trying to understand what was happening. This helps!

Reply:
Sure, Ask me anytime related to snaps. Thanks.